src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
@LiHaoTan Perhaps you can open a new issue to discuss this so that this PR can continue. For now, I feel `/*noinspection CssNoGenericFontName*/` should be removed. The only inspections we currently enforce are those performed by our current static analysis tools. If you want to suggest enforcing other inspections, it can be put up for discussion in the issue tracker.